src/PlaywrightSharp/DialogType.cs

```diff
@@ -6,27 +6,26 @@ namespace PlaywrightSharp
     /// Dialog type.
     /// </summary>
     /// <seealso cref="IDialog"/>
-    public enum DialogType
+    public static class DialogType
```

Comment:
What do you think about this @avodovnik?

Reply:
Looks good to me. I would consider moving it into `/Contracts/Models` though, despite not being generated.